tests/test_load_config.py

```diff
@@ -118,3 +120,26 @@ def test_pipeline_names():
                 },
             }
         )
+
+
+def test_french_defaults_and_override():
+    subj_tags_potatoes = ["russet", "yukon_gold"]
+    cfg = AzimuthConfig(
+        **{
+            **DEFAULT_CONFIG,
+            **{"language": "fr", "syntax": {"subj_tags": subj_tags_potatoes}},
+        }
```

Comment:
Let's clean up a couple fo `*`s.
```suggestion
        **DEFAULT_CONFIG,
        language="fr",
        syntax={"subj_tags": subj_tags_potatoes}},
```

Reply:
`*` suggestion!

😆

Reply:
@JosephMarinier Does the same logic apply to lines 66 and 81?

Reply:
It doesn't work...it says, "a variable annotation cannot be combined with tuple unpacking. I am doing what you suggested minus the second `}` on line 130 (since it doesn't match anything before it anymore).

Reply:
> Does the same logic apply to lines 66 and 81?

Yes. You have probably noticed that this was addressed on `main`, and just got merged into your branch.

You're correct, I included an extra `}` by mistake.

I am not able to reproduce the error you mention. I'll react out so we take a look together.

Reply:
All sorted, my syntax mistake, thanks!